src/Elmish.WPF.Tests/BindingVmHelpersTests.fs

```diff
@@ -0,0 +1,144 @@
+﻿module BindingVmHelpersTests.M
+
+open System
+
+open Xunit
+open Hedgehog
+open Swensen.Unquote
+
+open Elmish.WPF
+open Elmish.WPF.BindingVmHelpers
+
+
+
+module Initialize =
+
+  let checker<'t when 't : equality> (g: Gen<'t>) =
+    Property.check <| property {
+      let! m1 = g
+
+      let binding =
+        BindingData.OneWay.id
+        |> BindingData.mapModel box
+
+      let vmBinding =
+        Initialize(LoggingViewModelArgs.none, "Nothing", (fun _ -> failwith "Should not call get selected item"))
+          .Recursive(m1, ignore, (fun () -> m1), binding)
+
+      let result =
+        match vmBinding with
+        | Some (BaseVmBinding (OneWay b)) -> b
```

Comment:
I don't like these kinds of tests.  I think this code is too tightly coupled to the implementation.  This code is like the tests in `BindingTests.fs`, which I also don't like.  I like the tests in `DynamicViewModelTests.fs`.

Do you think you can write this test with less coupling to the implementation?

Reply:
We can just drop the initialize test. The others don't reach in so deeply, and all depend on initialize to work.

Reply:
It doesn't really make sense to test initialize tbh.

The other thing is I might refactor my `StaticHelper` to be the backing of these types even for `DynamicViewModel` (obviously my helper would need a rename), which would provide a cleaner interface to test (although this one is already decently clean).

Reply:
For the record, the initialize test was removed